plugins/backup/veeam/src/main/java/org/apache/cloudstack/backup/veeam/VeeamClient.java

```diff
@@ -303,7 +303,7 @@ private boolean checkTaskStatus(final HttpResponse response) throws IOException
                         String type = pair.second();
                         String path = url.replace(apiURI.toString(), "");
                         if (type.equals("RestoreSession")) {
-                            return checkIfRestoreSessionFinished(type, path);
+                            checkIfRestoreSessionFinished(type, path);
```

Comment:
if so, we could change the return type to "void" instead of "boolean"

Reply:
The boolean is used by other methods, do we have a reason to change it to void?

Reply:
I checked the code, it looks `checkIfRestoreSessionFinished` method is only used here (line 348)

Reply:
I misinterpreted the original comment, I've removed the return.